src/drivers/bitbucket_cloud.js

```diff
@@ -107,6 +167,18 @@ class BitBucketCloud {
 
     return await response.json();
   }
+
+  get sha() {
+    return BITBUCKET_COMMIT;
+  }
+
+  get branch() {
+    return BITBUCKET_BRANCH;
+  }
+
+  get user_email() {}
+
+  get user_name() {}
```

Comment:
are these meant to be blank?

Reply:
BB is not offering them

Reply:
What's the purpose of this `user_name`? Is it the person who started the action/pipleine? Is it the bot user? What do we use it for when it's defined? Why don't we need it defined?

Reply:
Its meant to be the CI user.

Reply:
Ok, so what do we use it for when it's defined?

Reply:
Actually we just only use it t setup the remote in Gitlab and previously was the "specific-by-vendor" olivaw